integration-tests/metrics-expected-warnings.txt

```diff
@@ -0,0 +1,60 @@
+metrics-collectd/src/main/java/io/dropwizard/metrics5/collectd/CollectdReporter.java:[175,30] [Slf4jLoggerDeclaration] SLF4J logger declarations should follow established best-practices
+metrics-collectd/src/main/java/io/dropwizard/metrics5/collectd/CollectdReporter.java:[306,57] [rawtypes] found raw type: Gauge
+metrics-core/src/main/java/io/dropwizard/metrics5/CsvReporter.java:[182,30] [Slf4jLoggerDeclaration] SLF4J logger declarations should follow established best-practices
```

Comment:
So with the `-XepOpt:Slf4jLoggerDeclaration:CanonicalStaticLoggerName=LOGGER` flag we get these warnings, because the patch mode still renames the fields from `LOGGER` to `LOG`. I must be overlooking something, but I don't see how. Will need to have look ~tomorrow with a fresh set of eyes.

Reply:
Okay, this appears to be a bug in Error Prone. [This diff](https://github.com/google/error-prone/compare/master...PicnicSupermarket:error-prone:sschroevers/unconditionally-apply-overrides) fixes it. Will try to write a test case before submitting upstream.

Reply:
Alright, I filed google/error-prone#4699.